cfgov/login/urls.py

```diff
@@ -33,6 +33,12 @@
 else:
     urlpatterns = [
         re_path(r"^login/$", LoginView.as_view(), name="cfpb_login"),
```

Comment:
How would you feel about, as part of this change, deprecating the above line and instead adding an Apache redirect that redirects from /login/ to /admin/login/? We don't appear to use the reverse of "cfpb_login" anywhere, so that is probably safe to remove as well. Otherwise we're defining two different Django URL patterns that point to the same place.

It would also be a step towards moving us away from our bespoke/custom use of /login/ as the login URL in favor of Wagtail's standard /admin/login/ - we could pair this with changing [`settings.LOGIN_URL`](https://github.com/cfpb/consumerfinance.gov/blob/67253be1bac2de3a37b76ff65ee4194a47287be4/cfgov/cfgov/settings/base.py#L422). I realize that might expand the scope of this PR and maybe you'd prefer to keep this more targeted.

Reply:
Maybe... I'd want to make sure it doesn't break the expectations around SAML2 auth (and remember how that's structured).

Reply:
Done, sort of, in [0e27746](https://github.com/cfpb/consumerfinance.gov/pull/7637/commits/0e27746db61b336364d88741619035dd565334e1). That commit message describes how it is expected to work now.